library/src/main/java/org/apache/apex/malhar/lib/function/Function.java

```diff
@@ -35,7 +35,7 @@
  * @since 3.4.0
  */
 @InterfaceStability.Evolving
-public interface Function
+public interface Function extends java.io.Serializable
```

Comment:
Would kryo's @DefaultSerializer work for interfaces? If so, maybe we should add it here?